test/AssetManager.js

```diff
@@ -100,8 +100,27 @@ describe('AssetManager', function () {
 
     it('should be able to inactivate Job', async function () {
       await assetManager.setAssetStatus(false, 5);
-      const job = await assetManager.jobs(5);
-      assert(job.active === false);
+      const isActive = await assetManager.getActiveStatus(5);
+      assert(isActive === false);
+    });
+
+    it('should be able to get a job', async function () {
+      const job = await assetManager.getJob(1);
+      assertBNEqual(job.active, 'true', 'job should be active');
+      assertBNEqual(job.repeat, 'true', 'repeat should be true for job 1');
+      assertBNEqual(job.name, 'test', 'job name should be "test"');
+      assertBNEqual(job.selector, 'selector', 'job selector should be "selector"');
+      assertBNEqual(job.url, 'http://testurl.com', 'job url should be "http://testurl.com"');
+    });
+
+    it('should not be able to create collection with incative jobs', async function () {
```

Comment:
All looks good, just correct this ```incative``` to ```inactive``` throughout the code

Reply:
+1

Reply:
ok